reg_tests/chgres_cube/driver.dell.sh

```diff
@@ -27,8 +27,8 @@ module load EnvVars/1.0.2
 module load ips/18.0.1.163
 module load impi/18.0.1
 module load lsf/10.1
-module use /usrx/local/dev/modulefiles
-module load NetCDF/4.5.0
+module use /usrx/local/nceplibs/dev/NCEPLIBS/modulefiles
+module load netcdf_parallel/4.7.4
```

Comment:
why are modules being loaded in the driver.XXX.sh? Isn't sourcing the machine appropriate modulefile enough?
The modulefile should load all the modules required.

Reply:
That is a good point.  I will make that change.